src/System.Net.Requests/tests/WebRequestTest.cs

```diff
@@ -81,6 +81,115 @@ public void Create_InvalidWebRequestUriScheme_Throws(string scheme)
+            Assert.Throws<NotImplementedException>(() => request.Method);
+            Assert.Throws<NotImplementedException>(() => request.Method = null);
+            Assert.Throws<NotImplementedException>(() => request.RequestUri);
+            Assert.Throws<NotImplementedException>(() => request.Headers);
+            Assert.Throws<NotImplementedException>(() => request.Headers = null);
+            Assert.Throws<NotImplementedException>(() => request.ContentLength);
+            Assert.Throws<NotImplementedException>(() => request.ContentLength = 0);
+            Assert.Throws<NotImplementedException>(() => request.ContentType);
+            Assert.Throws<NotImplementedException>(() => request.ContentType = null);
+            Assert.Throws<NotImplementedException>(() => request.Credentials);
+            Assert.Throws<NotImplementedException>(() => request.Credentials = null);
+            Assert.Throws<NotImplementedException>(() => request.Timeout);
+            Assert.Throws<NotImplementedException>(() => request.Timeout = 0);
+            Assert.Throws<NotImplementedException>(() => request.UseDefaultCredentials);
+            Assert.Throws<NotImplementedException>(() => request.UseDefaultCredentials = true);
+            Assert.Throws<NotImplementedException>(() => request.GetRequestStream());
+            Assert.Throws<NotImplementedException>(() => request.GetResponse());
+            Assert.Throws<NotImplementedException>(() => request.BeginGetResponse(null, null));
+            Assert.Throws<NotImplementedException>(() => request.EndGetResponse(null));
+            Assert.Throws<NotImplementedException>(() => request.BeginGetRequestStream(null, null));
+            Assert.Throws<NotImplementedException>(() => request.EndGetRequestStream(null));
+            Assert.Throws<NotImplementedException>(() => request.Abort());
+            Assert.Throws<NotImplementedException>(() => request.PreAuthenticate);
+            Assert.Throws<NotImplementedException>(() => request.PreAuthenticate = true);
+            Assert.Throws<NotImplementedException>(() => request.Proxy);
+            Assert.Throws<NotImplementedException>(() => request.Proxy = null);
+        }
+
+        public void GetSystemWebProxy_NoArguments_ExpectNotNull()
+        {
+            IWebProxy webProxy = WebRequest.GetSystemWebProxy();
+            Assert.NotNull(webProxy);
+        }
+
+        [Fact]
+        public void RegisterPrefix_PrefixOrCreatorNull_ThrowsArgumentNullException()
+        {
+            Assert.Throws<ArgumentNullException>(() => WebRequest.RegisterPrefix(null, new FakeRequestFactory()));
+            Assert.Throws<ArgumentNullException>(() => WebRequest.RegisterPrefix("http://", null));
+        }
+
+        [Fact]
+        public void RegisterPrefix_HttpWithFakeFactory_Success()
+        {
+            bool success = WebRequest.RegisterPrefix("sta://", new FakeRequestFactory());
+            Assert.True(success);
```

Comment:
Because "sta://" gets transformed to "sta:///" when calling it in `Uri.TryCreate(prefix, UriKind.Absolute, out tempUri)`